bindings/protocols/modbus/index.html

```diff
@@ -70,7 +70,7 @@ <h2>URL format</h2>
         Considering that in Web of Things context all protocol bindings are required to at least support the Internet Protocol, for the Modbus protocol <code>modbus+tcp://</code> 
         was selected as the only valid URL scheme. The following shows the typical structure of an URL of the Modbus protocol:
         <pre>
-            modbus+tcp://{address}:{port}/{?unitID}
+            modbus+tcp://{address}:{port}/{unitID}/{address}?quantity={?quantity}
```

Comment:
I would prefer this version since it would meet

-  the human-readable aspect
-  meets the single address approach with href
-  would be clearer to define REG expressions for validation
-  would be clearer to define search queries (e.g., I like to find all properties that have quantity==4)

Reply:
Although I agree that this version improves human-readability, I think it does not strictly comply with the semantics of URI components. In particular, I cite the definitions of [Path](https://www.rfc-editor.org/rfc/rfc3986#section-3.3) and [Query](https://www.rfc-editor.org/rfc/rfc3986#section-3.4) components from the RFC:

**Path**
> The path component contains data, usually organized in hierarchical
   form, that, along with data in the non-hierarchical query component
   ([Section 3.4](https://www.rfc-editor.org/rfc/rfc3986#section-3.4)), serves to identify a resource within the scope of the
   URI's scheme and naming authority (if any)
   
**Query**
> The query component contains non-hierarchical data that, along with
   data in the path component ([Section 3.3](https://www.rfc-editor.org/rfc/rfc3986#section-3.3)), serves to identify a
   resource within the scope of the URI's scheme and naming authority
   (if any).
   
In my understanding, the unitID and address properties are clearly hierarchical in nature as they define resources inside the space controlled by the base address. Moving this information in the query component will cause a loss of the real structure and it will mess with the URI arithmetic. In practise, I don't think we can define a common base address with the unitID anymore.